### ISSUE TITLE
R (≥ 3.0.2) dependence

### ISSUE THREAD

**Issue by jlegewie** [tidyverse/dplyr#185]
Hi,

`dplyr` seems to strictly depend on R 3.0.2. First, I get the common message `package ‘dplyr’ is not available (for R version 3.0.1)`. After that, I can usually install packages from source using `install.packages('dplyr_0.1.tar.gz',repos=NULL,type='source')`. `dplyr`, however doesn't play along and throughs this error at me: `ERROR: this R is version 3.0.1, package 'dplyr' requires R >= 3.0.2`

Not sure about the technical background but is this strict dependence necessary? Why not 3.0.1? There are many environments, which the user can't easily control and update to the most recent R version. In this case, I am working on a high performance cluster with preinstalled software. 

Thanks for the great package!

**Comment by jbmartin** [comment]
@jlegewie's suggestion worked for me on R 3.1.1, but I only had to change the Depends line in the DESCRIPTION file.

**Comment by gmxwilliam** [comment]
@jlegewie 

I have the same issue to install dplyr in R 3.0.1. Followed your instruction, I still can't install it successfully. I have specified the path of the Rcpp include directory.
Below is the output message, do you have any suggestion? Thanks.

* installing *source* package ‘dplyr’ ...
files ‘DESCRIPTION’, ‘src/Makevars’ have the wrong MD5 checksums
** libs
g++ -m64 -I/usr/include/R -DNDEBUG -I/home/windapp/R/x86_64-redhat-linux-gnu-lib
rary/3.0/Rcpp/include -I/usr/local/include    -fpic  -O2 -g -pipe -Wall -Wp,-D_F
ORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -m
tune=generic  -c RcppExports.cpp -o RcppExports.o
In file included from RcppExports.cpp:5:
../inst/include/dplyr.h:11:32: error: boost/scoped_ptr.hpp: No such file or dire
ctory
../inst/include/dplyr.h:12:37: error: boost/functional/hash.hpp: No such file or
 directory
../inst/include/dplyr.h:18:39: error: boost/unordered_map.hpp: No such file or d
irectory
../inst/include/dplyr.h:19:39: error: boost/unordered_set.hpp: No such file or d
irectory
../inst/include/dplyr.h:24:25: error: tools/tools.h: No such file or directory
../inst/include/dplyr.h:58:43: error: dplyr/visitor_set/visitor_set.h: No such f
ile or directory
../inst/include/dplyr.h:59:45: error: dplyr/DataFrameVisitorsIndexSet.h: No such
 file or directory
../inst/include/dplyr.h:60:45: error: dplyr/DataFrameVisitorsIndexMap.h: No such
 file or directory
../inst/include/dplyr.h:61:30: error: dplyr/BoolResult.h: No such file or direct
ory
../inst/include/dplyr.h:63:33: error: dplyr/FullDataFrame.h: No such file or dir
ectory
../inst/include/dplyr.h:64:36: error: dplyr/GroupedDataFrame.h: No such file or
directory
../inst/include/dplyr.h:65:27: error: dplyr/tbl_cpp.h: No such file or directory

../inst/include/dplyr.h:66:31: error: dplyr/comparisons.h: No such file or direc
tory
../inst/include/dplyr.h:67:33: error: dplyr/VectorVisitor.h: No such file or dir
ectory
../inst/include/dplyr.h:68:32: error: dplyr/OrderVisitor.h: No such file or dire
ctory
../inst/include/dplyr.h:69:37: error: dplyr/VectorVisitorImpl.h: No such file or
 directory
../inst/include/dplyr.h:70:37: error: dplyr/DataFrameVisitors.h: No such file or
 directory
../inst/include/dplyr.h:71:36: error: dplyr/OrderVisitorImpl.h: No such file or
make: *** [RcppExports.o] Error 1
Error: compilation failed for package 'dplyr'

**Comment by natashasm** [comment]
I keep getting errors when I tried uploading dyplr; this has not happened until I updated my Rstudio to 3.4 then I switched back to 3.3 version. It is not still working. Would anyone be able to help out on this issue?

> library("plyr", lib.loc="/Library/Frameworks/R.framework/Versions/3.3/Resources/library")
> library("dplyr", lib.loc="/Library/Frameworks/R.framework/Versions/3.3/Resources/library")

Attaching package: ‘dplyr.'

The following objects are masked from ‘package:plyr’:

    arrange, count, desc, failwith, id, mutate, rename, summarise, summarize

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> install.packages("dyplr")
Warning in install.packages :
  package ‘dyplr’ is not available (for R version 3.3.3)
![image](https://cloud.githubusercontent.com/assets/9164591/25765783/bb8566cc-31bc-11e7-9405-e6ab99087cca.png)
![image](https://cloud.githubusercontent.com/assets/9164591/25765791/ca705282-31bc-11e7-99b7-7d6a2590dacf.png)
